device-server/src/main/kotlin/com/badoo/automation/deviceserver/host/SimulatorsNode.kt

```diff
@@ -87,7 +88,9 @@ class SimulatorsNode(
 
             logger.debug(simLogMarker, "Will create simulator $ref")
 
-            val simulator = simulatorFactory.newSimulator(ref, remote, fbSimctlDevice, ports, deviceSetPath, wdaRunnerXctest, concurrentBoot, desiredCaps.headless, fbSimctlDevice.toString())
+            val simulator = simulatorFactory.newSimulator(ref, remote, fbSimctlDevice, ports, deviceSetPath,
+                    wdaRunnerXctest, concurrentBoot, desiredCaps.headless, desiredCaps.useWda, fbSimctlDevice.toString())
+            simulator.initialEnvironmentVariables.putAll(initialEnvironmentVariables)
```

Comment:
See my comment about `initialEnvironmentVariables` above. I think it should be removed.

Reply:
Done.
#34